bpf/lib/conntrack.h

```diff
@@ -766,19 +746,13 @@ static __always_inline int ct_create4(const void *map_main,
 	entry.lb_loopback = ct_state->loopback;
 	entry.node_port = ct_state->node_port;
 	entry.dsr = ct_state->dsr;
+	entry.ifindex = ct_state->ifindex;
 
-	/* We need to store the backend_id (points to a svc v2 endpoint), while
-	 * handling migration for users upgrading from prior releases. where
-	 * the "ct_entry.backend_id" field was used for legacy services.
-	 *
-	 * Previously, the rx_bytes field was not used for entries with
+	/* Previously, the rx_bytes field was not used for entries with
 	 * the dir=CT_SERVICE (see GH#7060). Therefore, we can safely abuse
-	 * this field to save the backend_id. The hack will go away once we stop
-	 * supporting the legacy svc (in v1.6 we will zero the backend_id
-	 * field, in v1.7 we can remove the rx_bytes hack).
```

Comment:
I guess we ended up sticking with the rx_bytes hack to avoid resizing the conntrack entries...?

Reply:
Unfortunately, yes, as resizing the conntrack entries is not straightforward at all.